diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter2/KdocMethodsTest.kt

```diff
@@ -272,6 +272,81 @@ class KdocMethodsTest : LintTestBase(::KdocMethods) {
         )
     }
 
+    @Test
+    @Tag(WarningNames.KDOC_WITHOUT_THROWS_TAG)
+    fun `No warning when catch without matching throw`() {
```

Comment:
What does this test name mean?

Reply:
Changed: `No warning when throw has matching catch`